rca/static_src/sass/components/_sitewide-alert.scss

```diff
@@ -0,0 +1,55 @@
+.sitewide-alert {
+    @include z-index(modal);
+    display: none;
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100vw;
+    height: 100vh;
+    overflow: auto;
+    background-color: $color--black-80;
+
+    // Applied if cookie just added
+    &--inactive {
+        display: none;
+    }
+
+    &--active {
+        display: block;
+    }
+
+    &__modal {
+        @include z-index(modal-controls);
+        background-color: $color--white;
+        color: $color--black;
+        position: relative;
+        top: 50%;
+        transform: translateY(-50%);
+        width: 100%;
+        max-width: 500px;
+        margin: 0 auto;
+        padding: ($gutter * 1.5);
+    }
+
+    &__heading {
```

Comment:
I can't see a sitewide-alert__heading class in the html, guess this might not be needed in the CSS?

Reply:
Good spot Nick! Thanks :)